db/schema.rb

```diff
@@ -10,10 +10,16 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_09_08_172657) do
+ActiveRecord::Schema[7.0].define(version: 2023_09_11_165304) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "customers", force: :cascade do |t|
+    t.string "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```

Comment:
What you're missing here is your joins table (customer_items) which would have an `item_id` and a `customer_id`